packages/node_modules/@node-red/editor-client/src/js/ui/view-tools.js

```diff
@@ -368,6 +368,8 @@ RED.view.tools = (function() {
 
 
     function gotoNearestNode(direction) {
+        // Do not select a nearest node if move is active
+        if (RED.view.state() == 3) { return }
```

Comment:
```suggestion
        if (RED.view.state() === RED.state.MOVING_ACTIVE) { return }
```

Reply:
Right 😉

Reply:
I suspect there will be lots of cases where keyboard actions can trigger things to happen when they shouldn't. Not proposing we fall down the hole of finding/fixing them - no-one has complained about it other than this one specific issue.